at_functional_test/lib/secondary/base/config/config.yaml

```diff
@@ -9,8 +9,7 @@ root_server:
 
 # The @Protocol application log configurations
 log:
-  debug: true
-  trace: true
+  level: FINEST
```

Comment:
We might want to keep FINEST here for when we run tests.

Reply:
yes I think so